indicnlp/tokenize/sentence_tokenize.py

```diff
@@ -32,6 +31,19 @@
 ## pattern to check for presence of danda in text 
 CONTAINS_DANDA=re.compile(r'[\u0964\u0965]')
 
+def is_latin_or_numeric(character):
```

Comment:
Can't we use `str.isalnum()` instead?

Reply:
The logic here is that, if "." is preceded by Latin characters only then we don't want to not consider it as a sentence break (this is a specific case is for handling urls and emails). 

str.isalnum() is not restricted to just Latin characters, even if a Devanagari character is passed to str.isalnum() it returns True and therefore that cannot be used in this case for the above validation.

Reply:
Thanks for the explanation. Regarding that use case below, urls can have characters like +# etc. Try checking for the entire ASCII range instead of this regex.

Reply:
Test with this url, for instance: https://en.wikipedia.org/wiki/Chinese_softshell_turtle#Diet

Reply:
Currently, inappropriate sentence splits in cases or URLs/emails occur only due to the presence of "." in URLs / emails. The presence / absence of other characters like '#' in the URL do not cause any problems / do not invalidate the current check in place. Only the characters which precede or follow a "." are the ones which should be completely covered by our regex.

Acceptable characters for DNS domain names include :-
Allowed characters: DNS names can contain only alphabetic characters (A-Z), numeric characters (0-9), the minus sign (-), and the period (.). Period characters are allowed only when they're used to delimit the components of domain style names.

Accordingly, I have updated the regex to ensure that all these characters are covered.

Source :- https://learn.microsoft.com/en-us/troubleshoot/windows-server/identity/naming-conventions-for-computer-domain-site-ou